docs/source/notebooks/RatTumorExample.ipynb

```diff
@@ -0,0 +1,448 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Rat Tumors"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Running on PyMC3 v3.4.1\n"
+     ]
+    }
+   ],
+   "source": [
+    "%matplotlib inline\n",
+    "import pymc3 as pm\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "import seaborn as sns\n",
+    "import pandas as pd\n",
+    "import pymc3.distributions.transforms as tr\n",
+    "import theano.tensor as tt\n",
+    "from scipy.special import gammaln\n",
+    "\n",
+    "\n",
+    "\n",
+    "plt.style.use('seaborn-darkgrid')\n",
+    "print('Running on PyMC3 v{}'.format(pm.__version__))\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This short tutorial demonstrates how to use pymc3 to do inference for the rat tumour example found in chapter 5 of *Bayesian Data Analysis 3rd Edition*.  Readers should already be familliar with the pymc3 api.\n",
+    "\n",
+    "Suppose we are interested in the probability that a lab rat develops endometrial stromal polyps.  We have data from 71 previously performed trials and would like to use this data to perform inference.\n",
+    "\n",
+    "The authors of BDA3 choose to model this problem heirarchically.  Let $y_i$ be the number of lab rats which develop endometrial stromal polyps out of a possible $n_i$.  We model the number rodents which develop endometrial stromal polyps as binomial\n",
+    "\n",
+    "$$ y_i \\sim \\operatorname{Bin}(\\theta_i;n_i)$$\n",
+    "\n",
+    "allowing the probability of developing an endometrial stromal polyp (i.e. $\\theta_i$) to be drawn from some population distribution.  For analytical tractability, we assume that $\\theta_i$ has Beta distribution\n",
+    "\n",
+    "$$ \\theta_i \\sim \\operatorname{Beta}(\\alpha, \\beta)\\>. $$\n",
+    "\n",
+    "We are free to specify a prior distribution for $\\alpha, \\beta$.  We choose a weakly informative prior distribution to reflect our ignorance about the true values of $\\alpha, \\beta$.  The authors of BDA3 choose the joint hyperprior for $\\alpha, \\beta$ to be\n",
+    "\n",
+    "$$ p(\\alpha, \\beta) \\propto (\\alpha + \\beta) ^{-5/2} \\>.$$\n",
```

Comment:
The math rendering looks a bit odd:
<img width="203" alt="screen shot 2018-06-27 at 8 06 38 am" src="https://user-images.githubusercontent.com/12952641/41955740-9bf1cc42-79e1-11e8-9c49-2406e4809d97.png">